Jenkinsfile

```diff
@@ -43,7 +43,6 @@ pipeline {
         sh "composer create-project magento/community-edition magento"
         sh "cd magento && composer config minimum-stability dev"
         sh "cd magento && composer config prefer-stable true"
-        sh "cd magento && composer config repositories.2 git https://github.com/${env.CHANGE_FORK}/nosto-magento2"
```

Comment:
Can you revert this line since the issue was a temporary glitch in Jenkins?

Reply:
Reverted